docs/en/advanced_guides/models.md

```diff
@@ -1 +1,119 @@
 # Models
+
+We usually define a neural network in a deep learning task as a model, and this model is the core of an algorithm. [MMEngine](https://github.com/open-mmlab/mmengine) abstracts a unified model [BaseModel](https://github.com/open-mmlab/mmengine/blob/main/mmengine/model/base_model/base_model.py#L16) to standardize the interfaces for training, testing and other processes. All models implemented by MMSegmentation inherit from `BaseModel`, but we have re-implemented some of the interfaces for semantic segmentation tasks.
+
+## Common components
+
+### Segmentor
+
+In MMSegmentation, we abstract the network architecture as a **Segmentor**, it is a model that contains all components of a network. We have already implemented **EncoderDecoder** and **CascadeEncoderDecoder**, which typically consist of **backbone**, **decode_head**, **auxiliary_head**.
```

Comment:
it would be better to add some explanation about `auxiliary_head`, and the difference between `auxiliary_head` and `decode_head` as users always have questions about `auxiliary_head` function.